src/main/java/nl/bioinf/minorapplicationdesign/ontpillen/model/data_storage/DrugSubstance.java

```diff
@@ -47,22 +51,30 @@ public void setName(String name) {
     }
 
     public List<String> getBrandNames() {
-        return brandNames;
+        return Collections.unmodifiableList(brandNames);
     }
 
     public List<String> getDescription() {
-        return description;
+        return Collections.unmodifiableList(description);
     }
 
     public List<String> getSideEffects() {
-        return sideEffects;
+        return Collections.unmodifiableList(sideEffects);
     }
 
     public List<String> getUseIndications() {
-        return useIndications;
+        return Collections.unmodifiableList(useIndications);
     }
 
     public List<StopIndication> getStopIndications() {
-        return stopIndications;
+        return Collections.unmodifiableList(stopIndications);
+    }
+
+    public List<String> getInteractions() {
+        return Collections.unmodifiableList(interactions);
+    }
+
+    public void setInteractions(List<String> interactions) {
```

Comment:
Omdat interactions een lijst is is het misschien logischer een addInteraction te hebben die iets toevoegt aan de lijst.